app/controllers/games_controller.rb

```diff
@@ -3,7 +3,12 @@ class GamesController < ApplicationController
 
   # GET /games or /games.json
   def index
-    @games = Game.all
+    allowed_sort_columns = %w[id created_at state]
+
+    sort_column = params[:sort].presence_in(allowed_sort_columns) || :id
+    sort_order = params[:order] == 'desc' ? 'desc' : 'asc'
```

Comment:
I don't think this is the place for these variables. I will add more details in the review comment